testsuite/integration/web/web-enable-elytron.cli

```diff
@@ -1,6 +1,6 @@
 embed-server --server-config=standalone.xml
 
-/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
+#/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
```

Comment:
I can put it back. It will have no effect on the tests though. I admit that it ensures that there is at least some security domain referencing Elytron in the Undertow subsystem, even though it is configured in server setup tasks themselves anyway.